samples/boards/nrf52/mesh/onoff_level_app/src/board.h

```diff
@@ -0,0 +1,28 @@
+/* board.h - Board-specific hooks */
```

Comment:
This file seems unnecessary since you're only targetting the nRF52 (i.e. a single board). I know it's there in other samples like samples/bluetooth/mesh, however those are designed to be adapted potentially on multiple different boards.